modules/connectivity/locals.tf

```diff
@@ -1459,6 +1463,7 @@ locals {
     azure_database_for_mariadb_server    = ["privatelink.mariadb.database.azure.com"]
     azure_database_for_mysql_server      = ["privatelink.mysql.database.azure.com"]
     azure_database_for_postgresql_server = ["privatelink.postgres.database.azure.com"]
+    azure_databricks                     = ["privatelink.azuredatabricks.net"]
```

Comment:
I feel like there were more private dns zones added in the upstream policy. Should they all have been added here?

Reply:
No good catch!

Reply:
Added